.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,68 @@
+name: 🐞 Bug
+description: Create a report to help us improve
+title: "<title>"
+labels: ["Bug"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
```

Comment:
Will this checkbox be visible in the resulting issue?

Would it be better just to have a preamble mentioning to search for similar issues, but without some input?

Example: https://github.com/containers/podman/issues/new/choose

Reply:
> Will this checkbox be visible in the resulting issue?

yes

> Would it be better just to have a preamble mentioning to search for similar issues, but without some input?

I proceeded from the premise that it should be an obvious first action that requires minimal effort (just a click)

> Example: https://github.com/containers/podman/issues/new/choose

**Q:** do you mean [this](https://github.com/containers/podman/issues/new?assignees=&labels=kind%2Fbug%2Ctriage-needed&projects=&template=bug_report.yaml) bug template?

Reply:
Yeah, that's the template I was referring to.

I think it's good to push people to search for similar issues, but I wouldn't want any output related to this to end up in the issue itself. I think it would be distracting.

Reply:
Will be addressed in the follow-up PR

Reply:
Fixed in https://github.com/kamu-data/kamu-cli/pull/422